tests/avifincrtest_helpers.c

```diff
@@ -331,11 +343,11 @@ avifBool decodeIncrementally(const avifRWData * encodedAvif, const avifImage * r
     // Parsing is not incremental.
     avifResult parseResult = avifDecoderParse(decoder);
     while (parseResult == AVIF_RESULT_WAITING_ON_IO) {
-        if (availableEncodedAvif.size >= encodedAvif->size) {
+        if (data.available.size >= data.fullSize) {
             printf("ERROR: avifDecoderParse() returned WAITING_ON_IO instead of OK\n");
             goto cleanup;
         }
-        availableEncodedAvif.size = availableEncodedAvif.size + 1;
+        data.available.size = data.available.size + 1;
```

Comment:
Nit: This doesn't really matter. This can be written as data.available.size++. I was just wondering if it was deliberately written this way (perhaps for style reasons).

Reply:
You are right, I tend to avoid `++` for quantity-related vars to hint that it is not an iterator nor a counter. I would have used `+= 1` but I did not remember if it was valid in C I guess.